public/locales/en/common.json

```
@@ -9,11 +11,14 @@
   "developerTraits": "Developer traits",
   "failedLoadTraits": "Failed to load traits",
   "madeBy": "Made by the $t(title) community: ",
-  "owner": "Owner",
+  "daoGithubOrg": "Developer DAO GitHub organization",
+  "daoGithubRepo": "Developer DAO GitHub repository",
```

Would it make sense to break that down a bit more?


```suggestion
  "organization": "organization", 
  "repository": "repository",
  "daoGithubOrg": "$t(title) GitHub $t(organization)",
  "daoGithubRepo": "$t(title) GitHub $(repository)",
```

I feel like "organization" and "repository" are fairly common words in project docs, so it might make things easier in the future to translate them separately.

---

> A few suggestions/questions that I wanted to get your opinion on! No required changes though, we can merge if we feel like we don't need to worry about them

@with-heart thanks a lot! I'll make adjustment based in the feedbacks today.